e2e/podman/input/registry.hcl

```diff
@@ -0,0 +1,137 @@
+# Copyright (c) HashiCorp, Inc.
+# SPDX-License-Identifier: MPL-2.0
+
+# This job stands up a private container registry for use in e2e tests.
```

Comment:
idle pondering: I wonder if a general version of this might be nice for other things one day.

Reply:
we should at least add similar tests for the docker driver, but that's for another PR